spec/validators/outgoings_assessment/answers_validator_spec.rb

```diff
@@ -312,6 +341,30 @@
     end
   end
 
+  describe '#partner_income_tax_rate_complete?' do
+    let(:involvement_in_case?) { 'none' }
+
+    context 'when partner_income_tax_rate_above_threshold is present' do
+      before do
+        allow(record).to receive(:partner_income_tax_rate_above_threshold).and_return(true)
+      end
+
+      it 'returns true' do
+        expect(subject.partner_income_tax_rate_complete?).to be(true)
+      end
+    end
+
+    context 'when partner_income_tax_rate_above_threshold is nil' do
```

Comment:
Would it be possible for this value to be nil if the partner was part of the means assessment?

Reply:
Thinking save and come back later I guess